core/domain/acl_decorators.py

```diff
@@ -343,24 +343,89 @@ def test_can_access(self, **kwargs):
     return test_can_access
 
 
-def can_comment_on_feedback_thread(handler):
-    """Decorator to check whether the user can view feedback for a given
-    exploration.
+def can_create_feedback_thread(handler):
+    """Decorator to check whether the user can create a feedback thread.
     """
 
     def test_can_access(self, exploration_id, **kwargs):
-        """Checks if the user can view the exploration feedback.
+        """Checks if the user can create a feedback thread.
 
         Args:
-            exploration_id: str. The exploration id.
+            exploration_id: str. The ID of the exploration where the thread will
+                be created.
+            **kwargs: *. Keyword arguments.
+
+        Returns:
+            bool. Whether the user can create the feedback thread.
+        """
+        if exploration_id in feconf.DISABLED_EXPLORATION_IDS:
+            raise base.UserFacingExceptions.PageNotFoundException
+
+        exploration_rights = rights_manager.get_exploration_rights(
+            exploration_id, strict=False)
+        if rights_manager.check_can_access_activity(
+                self.user, exploration_rights):
+            return handler(self, exploration_id, **kwargs)
+        else:
+            raise self.UnauthorizedUserException(
+                'You do not have credentials to create exploration feedback.')
+    test_can_access.__wrapped__ = True
+
+    return test_can_access
+
+
+def can_view_feedback_thread(handler):
+    """Decorator to check whether the user can view a feedback thread.
+    """
```

Comment:
Might as well move the """ to end of previous line?

/cc @apb7

Reply:
Done

Reply:
I think the linter must have caught this?

Reply:
Actually it didn't.. Can't remember if I had a larger docstring where the linter caught it and then I edited the docstring but didn't move the """ up a line.